operator/config/crd/bases/forklift.konveyor.io_migrations.yaml

```diff
@@ -432,6 +432,9 @@ spec:
                     type:
                       description: Type used to qualify the name.
                       type: string
+                    vmFimware:
+                      description: virt-v2v ovf file
```

Comment:
need to update the description, here we can say that this is the firmware that was detected by virt-v2v while converting the guest